app/actions/claim.ts

```diff
@@ -185,9 +185,12 @@ async function handleSendBonk(chompResults: ChompResult[], address: string) {
   const treasurySolBalance = await getSolBalance(treasuryAddress);
   const treasuryBonkBalance = await getBonkBalance(treasuryAddress);
 
-  if (treasurySolBalance < 0.1 || treasuryBonkBalance < 10000000) {
+  if (
+    treasurySolBalance < 0.1 ||
+    (treasuryBonkBalance < 10000000 && treasuryBonkBalance > 0)
```

Comment:
Can you comment the logic here
```suggestion
    // getBonkBalance returns 0 for RPC errors, so we don't trigger Sentry if low balance is just RPC failure
    (treasuryBonkBalance < 10000000 && treasuryBonkBalance > 0)
```